src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -53,6 +53,10 @@
 ;;     Controls whether or not columns are reorderable.  When true, a reorder widget is presented
 ;;   :reorder-anchor (optional, default :left)
 ;;     Which side to anchor the reordering overlay.  Set to :right if placing the widget on the right side.
+;;   :reorder-style (optional, no default style)
+;;     Applies style properties to the displayed columns in the reorder widget
+;;   :reorder-prefix (optional, nil)
+;;     Prefixes the widget with the provided text
```

Comment:
Discussed this approach with David Mohs. This is the suggested short-term solution, adding in optional styling info that will be used when `Table` creates the column widget. Longer term solution is to refactor table to take an optional widget component and build one if it doesn't exist.